ideascube/monitoring/tests/test_migration_unique_serial.py

```diff
@@ -0,0 +1,71 @@
+from ideascube.tests.helpers import migration_test
+
+
+@migration_test(
+    migrate_from=[('monitoring', '0005_auto_20161027_0801')],
+    migrate_to=[('monitoring', '0006_unique_serial')]
+)
+def test_do_no_break_already_unique_serial(migration):
+    Specimen = migration.old_apps.get_model('monitoring', 'Specimen')
+    StockItem = migration.old_apps.get_model('monitoring', 'StockItem')
+
+    stockItem = StockItem()
+    stockItem.save()
+
+    for i in range(5):
+        specimen = Specimen(item_id=stockItem.id, barcode=str(i), serial=str(i)*5)
```

Comment:
The `*5` seems a bit gratuitous. `str(i)` would have been just fine.

Not asking you to change it now, it doesn't matter much. It's just that it makes it feel "special". I had to go back to the model to verify whether there was a minimum length. 😃